it_IT/code-of-conduct.md

```diff
@@ -0,0 +1,30 @@
+___
```

Comment:
> It will just replace [INSERIRE INDIRIZZO EMAIL] in the file by [sindresorhus@gmail.com](mailto:sindresorhus@gmail.com)

Sorry, what do you mean by this?